taginfo.json

```diff
@@ -1,4461 +1,6589 @@
-      },
-      {
-         "key": "bicycle",
-         "value": "use_sidepath",
-         "object_types": [
-            "node",
-            "way"
-         ],
-         "description": "Overrides access flags for bicycle.  Bicycle Routing allowed."
-      },
-      {
-         "key": "bicycle",
-         "value": "no",
-         "object_types": [
-            "node",
-            "way"
-         ],
-         "description": "Overrides access flags for bicycle.  Bicycle Routing not allowed."
-      },
-      {
-         "key": "bicycle",
-         "value": "permissive",
-         "object_types": [
-            "node",
-            "way"
-         ],
-         "description": "Overrides access flags for bicycle.  Bicycle Routing allowed."
-      },
-      {
-         "key": "bicycle",
-         "value": "destination",
-         "object_types": [
-            "node",
-            "way"
-         ],
-         "description": "Overrides access flags for bicycle.  Bicycle Routing allowed."
-      },
-      {
-         "key": "bicycle",
-         "value": "dismount",
-         "object_types": [
-            "node",
-            "way"
-         ],
-         "description": "Overrides access flags for bicycle.  Bicycle Routing allowed."
-      },
-      {
-         "key": "bicycle",
-         "value": "lane",
-         "object_types": [
-            "node",
-            "way"
-         ],
-         "description": "Overrides access flags for bicycle.  Bicycle Routing allowed."
-      },
-      {
-         "key": "bicycle",
-         "value": "track",
-         "object_types": [
-            "node",
-            "way"
-         ],
-         "description": "Overrides access flags for bicycle.  Bicycle Routing allowed."
-      },
-      {
-         "key": "bicycle",
-         "value": "shared",
-         "object_types": [
-            "node",
-            "way"
-         ],
-         "description": "Overrides access flags for bicycle.  Bicycle Routing allowed."
-      },
-      {
-         "key": "bicycle",
-         "value": "shared_lane",
-         "object_types": [
-            "node",
-            "way"
-         ],
-         "description": "Overrides access flags for bicycle.  Bicycle Routing allowed."
-      },
-      {
-         "key": "bicycle",
-         "value": "sidepath",
-         "object_types": [
-            "node",
-            "way"
-         ],
-         "description": "Overrides access flags for bicycle.  Bicycle Routing allowed."
-      },
-      {
-         "key": "bicycle",
-         "value": "share_busway",
-         "object_types": [
-            "node",
-            "way"
-         ],
-         "description": "Overrides access flags for bicycle.  Bicycle Routing allowed."
-      },
-      {
-         "key": "bicycle",
-         "value": "none",
-         "object_types": [
-            "node",
-            "way"
-         ],
-         "description": "Overrides access flags for bicycle.  
```

Comment:
looks like it is missing the `:forward` and `:backward` keys to several like this